products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,233 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - this tutorial could contain Game of Thrones ***spoilers***.  
+
+Game of Thrones is an American fantasy drama television series created by David Benioff and D. B. Weiss for HBO.
+It is an adaptation of A Song of Ice and Fire, George R. R. Martin's series of fantasy novels,  the first of which is
+A Game of Thrones. The Game of Thrones world is full of interesting characters, locations, scenarios, unexpected deaths, and plot twists.
+
+Even though, the COVID-19 pandemic hit the entire world in 2020 and is now starting to become one of the worst
+years in recent history, 2019 was also a huge disappointment to all the Game of Thrones fans. According to IMDb ratings, a seven-year 
+build-up resulted in a poorly written ending of the last season and ruined the ending of one of the most popular shows on the planet.  
+Nonetheless, if you want to know how many characters would have survived if Jon Snow had stayed dead, which House had the best Kill/Death Ratio
+or who was the biggest traitor in the show, you came to the right place!
+
+### Data Model
+
+Although the Game of Thrones TV show is based on series of books, our graph database contains only characters from 
+previously mentioned TV shows as books are still not finished. This tutorial would not be possible with data analyst David 
+Murphy who shared his collection of on-screen deaths on [this link](https://data.world/datasaurusrex/game-of-thones-deaths).We won't we working with kills and deaths that happened off-screen or were 
+tied to undeads (wraiths). The dataset we used was slightly modified, columns "Episode Name" and "IMDb Rating" were added.
+
+The model consists of the following nodes:
+* a `Character` node has a "name" attribute corresponding to the character's name (e.g. `"Jon Snow"`)
+* an `Allegiance` node has a "name" attribute corresponding to the house name the character 
+    is loyal to (e.g.  `"House Stark"`)
+* a `Death` node has an "order" attribute corresponding to the order in which the death happened 
+    in show (e.g. `602`)
+* an `Episode` node has a "number" attribute corresponding to the number of episodes (e.g. `10`), 

```

Comment:
Might as well use \`number\` instead of "number" so the markdown can render code.